content/en/blog/falco-0-38-0/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Introducing Falco 0.38.0
+date: 2024-05-22 # TODO set the correct date
+author: Luca Guerra
+slug: falco-0-38-0
+tags: ["Falco","Release"]
+---
+
+Dear Falco Community, today we are happy to announce the release of Falco 0.38.0! This is the first Falco release since its [graduation](/blog/falco-graduation/) within the CNCF, and, as usual, brings many improvements and features alongside some pretty big changes in its configuration mechanism.
+
+This release brings an easier to use mechanism to install and configuring your drivers, new rule language features, better support for Falco metrics and many more improvements.
+
+During this release cycle, we merged more than 100 PRs on Falco and more than 180 PRs for libs and drivers, version 0.17.0 and version 7.1.1 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
```

Comment:
```suggestion
During this release cycle, we merged more than 100 PRs on Falco and more than 180 PRs for libs and drivers, version 0.17.0 and version 7.2.0 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
```

Reply:
Thanks! 😎